### ISSUE TITLE
Fonctionnement de la mosaique loisirs culturels illogique

### ISSUE THREAD

**Issue by florianpanchout** [incubateur-ademe/nosgestesclimat#2073]
Lorsque j'arrive sur cette mosaïque tout est décoché. Si je coche "Musées et monuments" j'ai "-75kg" qui s'affiche. Si je le décoche ensuite j'ai "-76kg" qui s'affiche. Si je le recoche j'ai "-1kg".

**Comment by Clemog** [comment]
Je pense que c'est logique : 
- Première étape : rien n'est coché mais une valeur par défaut est attribuée (implicitement)
- Je clique sur Musées et monuments -> -75kg par rapport à la valeur par défaut 'cachée'
- Je décoche mais j'ai fait un choix utilisateur (coché puis décoché) donc on ne reprend plus la valeur par défaut : rien n'est coché: les loisirs culturels, c'est 0. D'ailleurs, le bouton de la mosaïque, n'est plus "Je ne sais pas" mais "Suivant"
- Si je recoche, j'ai +1kg, tu es sur du -1 ?

**Comment by florianpanchout** [comment]
Est-ce qu'on retrouve ailleurs ce fonctionnement de valeur par défaut cachée ? (elles sont visibles sur les autres mosaïques par exemple). Est-ce qu'il n'y a pas un problème de cohérence ? (en plus d'un problème d'UX)

**Comment by Clemog** [comment]
Sur toutes les mosaïques de sélection c'est le cas en principe

**Comment by florianpanchout** [comment]
Yes j'ai saisi (enfin je crois) la nécessité d'avoir une valeur par défaut. Mais pourquoi cacher la sélection faite par défaut sur 3 questions en pratique (chauffage, hébergement vacances et pratiques culturelles) et l'afficher sur toutes les autres ? Pourquoi n'a t'on pas décidé de cacher la valeur par défaut sur Animaux de compagnie par exemple ?

**Comment by Clemog** [comment]
Car on n'avait pas le retour sur les mosaïques numériques

**Comment by JuliePouliquen** [comment]
À fond pour une amélioration du fonctionnement des mosaïques :smile: https://github.com/incubateur-ademe/nosgestesclimat-site/issues/1311
